### ISSUE TITLE
ImageService context definition uses / but prefix uses #

### ISSUE THREAD

**Issue by azaroth42** [IIIF/api#1711]
The context has 

```
    "ImageService3": {
      "@id": "http://iiif.io/api/image/3/ImageService"
    }
```
But also

```
    "image": "http://iiif.io/api/image/3#",
```

Which is it?

**Comment by zimeon** [comment]
http://iiif.io/api/image/3#ImageService resolves the spec, http://iiif.io/api/image/3/ImageService is 404 --> I vote #

**Comment by azaroth42** [comment]
Editors agree #

**Comment by azaroth42** [comment]
Closed by #1718
